modules/install/pages/extensions.adoc

```diff
@@ -1,24 +1,316 @@
+
+* Start the JBoss EAP instance using the `standalone-ha.xml` to start clustering capabilities.
++
+[subs="+quotes,+macros"]
+----
+[vagrant@server bin]$ sudo -u jboss /bin/sh ./standalone.sh -c standalone-ha.xml
+----
+
+Press `ctrl+C` to stop the JBoss instance.
+
+* Start the JBoss EAP instance using the `standalone-ha.xml` to start messaging-activemq capabilities.
++
+[subs="+quotes,+macros"]
+----
+[vagrant@server bin]$ sudo -u jboss /bin/sh ./standalone.sh -c standalone-full.xml
+----
+
+Press `ctrl+C` to stop the JBoss instance.
+
+* Start the JBoss EAP instance using the `standalone-ha.xml` to start messaging-activemq and clustering capabilities.
++
+[subs="+quotes,+macros"]
+----
+[vagrant@server bin]$ sudo -u jboss /bin/sh ./standalone.sh -c standalone-full-ha.xml
+----
+
+Press `ctrl+C` to stop the JBoss instance.
+
+
+== Lab 2: Configuring Subsystems
+
+*Outcome*: In this lab, you customize the subsystem of JBoss EAP 8 standalone server by updating the log level using management console.
+
+* Navigate to the JBoss_Home directory.
++
+[subs="+quotes,+macros"]
+----
+[vagrant@server ~]$ cd /opt/EAP-8.0.0/bin/
+[vagrant@server bin]$
+----
+
+* Start the JBoss EAP instance using the default `standalone.xml` to start most commonly used subsystems.
++
+[subs="+quotes,+macros"]
+----
+[vagrant@server bin]$ sudo -u jboss /bin/sh ./standalone.sh -c standalone.xml
+----
 
-== Profiles
+* Then, navigate to the management console at `http://localhost:9990`. Use `admin` as username and `redhat123` as password.
 
-== Lab: Extensions, Subsystems and Profiles
+* Update the log level of the `CONSOLE` handler and set it to `DEBUG`.
 
+. Click Configuration at the top of the management console. In the first column, click Subsystems and then in the second column click Logging. Click Configuration in the third column, and then View.
++
+image::handle.png[align="center"]
 
-. Step 1
-. Step 2
-.. Step 2.a
-.. Step 2.b
-. Step 3
+. Click Handler at the first column, and then click Console Handler to access the configuration page for the Console handler.
++
+image::con.png[align="center"]
 
-* Unravel the actual file system layout and show some XML snippets and JBoss EAP CLI list commands
-* Some nice web console screenshots can go here...
-* Do not go too much into details about EAP CLI or web console here. Remember there is a separate chapter for EAP config...
-* Different modes - standalone, standalone-ha, domain etc,  but not too much detail. We have details in the advanced admin course
+. Select the CONSOLE row in the table. Click Edit. Change the Level from INFO to DEBUG. Then click Save.
++
+image::new.png[align="center"]
 
+* Stop the instance of the JBoss EAP listening on port 9990 by pressing `Ctrl+C` on the terminal where it is running and restart the instance to reflect the new changes. Observe the additional debug information provided by the server in the console that reflects the log level change made in the management console.
```

Comment:
restart not required to see logging changes. most subsystems can be hot reloaded

Reply:
modified